pages.id/common/pm2.md

```diff
@@ -0,0 +1,33 @@
+# pm2
+
+> Manajer proses untuk Node.js
+> Digunakan untuk manajemen log, pemantauan, dan pengaturan proses.
+> Informasi lebih lanjut: <https://pm2.keymetrics.io>.
+
+- Memulai prooses dengan nama yang dapat digunakan untuk operasi selanjutnya:
+
+`pm2 start {{app.js}} --name {{myapp}}`
+
+- Tampilkan daftar proses:
+
+`pm2 list`
+
+- Pantau semua proses:
+
+`pm2 monit`
+
+- Menghentikan sebuah proses:
+
+`pm2 stop {{myapp}}`
```

Comment:
I think we can update the English page to use `application_name` instead of `myapp`. Do you want to send the update?

Reply:
Ok sure, wait

Reply:
I'll use `nama_aplikasi` for this is Indonesian translation

Reply:
Done, please check it. Thanks!